### PR TITLE
Increase/Decrease Sound with mouse wheel on video hover

### PR DESCRIPTION
**Pull request by rguedes**
Increase/Decrease Sound with mouse wheel on video hover

### CODE REVIEW

File: renderer/views/player.js [hunk 1]
```diff
@@ -125,7 +131,7 @@ function renderMedia (state) {
   function onCanPlay (e) {
     var video = e.target
     if (video.webkitVideoDecodedByteCount > 0 &&
-      video.webkitAudioDecodedByteCount === 0) {
+        video.webkitAudioDecodedByteCount === 0) {
```
Comment (feross) [inline renderer/views/player.js]: Undo this change please

File: renderer/views/player.js [hunk 2]
```diff
@@ -195,7 +201,7 @@ function renderAudioMetadata (state) {
 function renderLoadingSpinner (state) {
   if (state.playing.isPaused) return
   var isProbablyStalled = state.playing.isStalled ||
-    (new Date().getTime() - state.playing.lastTimeUpdate > 2000)
+      (new Date().getTime() - state.playing.lastTimeUpdate > 2000)
```
Comment (feross) [inline renderer/views/player.js]: Undo this change please

File: renderer/views/player.js [hunk 3]
```diff
@@ -290,8 +296,8 @@ function renderPlayerControls (state) {
   var positionPercent = 100 * state.playing.currentTime / state.playing.duration
   var playbackCursorStyle = { left: 'calc(' + positionPercent + '% - 8px)' }
   var captionsClass = state.playing.subtitles.tracks.length === 0
-    ? 'disabled'
-    : state.playing.subtitles.enabled
+      ? 'disabled'
+      : state.playing.subtitles.enabled
```
Comment (feross) [inline renderer/views/player.js]: Same

File: renderer/views/player.js [hunk 4]
```diff
@@ -403,8 +409,8 @@ function renderPlayerControls (state) {
   var volume = state.playing.volume
   var volumeIcon = 'volume_' + (volume === 0 ? 'off' : volume < 0.3 ? 'mute' : volume < 0.6 ? 'down' : 'up')
   var volumeStyle = { background: '-webkit-gradient(linear, left top, right top, ' +
-    'color-stop(' + (volume * 100) + '%, #eee), ' +
-    'color-stop(' + (volume * 100) + '%, #727272))'
+  'color-stop(' + (volume * 100) + '%, #eee), ' +
+  'color-stop(' + (volume * 100) + '%, #727272))'
```
Comment (feross) [inline renderer/views/player.js]: Same

File: renderer/views/player.js [hunk 5]
```diff
@@ -10,6 +10,11 @@ var Bitfield = require('bitfield')
 var TorrentSummary = require('../lib/torrent-summary')
 var {dispatch, dispatcher} = require('../lib/dispatcher')
 
+// Handles volume change by wheel
+var handleVolumeWheel = function (e) {
+  dispatch('changeVolume', (-e.deltaY | e.deltaX) / 500)
+}
```
Comment (feross) [inline renderer/views/player.js]: Please use the function declaration style: `function handleVolumeWheel() {}` There is no need to change this function.

**Comment by feross** [comment]
I'm fine with this change. VLC does it and lots of people want it. However, this should be a one-line PR.

Can you fix this up so there aren't random whitespace and style changes? Thanks!

**Comment by rguedes** [comment]
Done. Sorry my first commit with JS standard style. My ide wasn't ready.

**Comment by feross** [comment]
Thanks - merged
